e2-studio-star-rx72n-firmware/build.sh

```
@@ -0,0 +1,134 @@
+    -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+    ..
+
+echo -e "${GREEN}✓ Configuration complete${NC}"
+
+# Build
+echo -e "${YELLOW}Building firmware...${NC}"
+
+if [ "$VERBOSE" = true ]; then
+    cmake --build . --verbose
+else
+    cmake --build . -- -j$(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)